docs/docs/intro.md

```diff
@@ -1,34 +1,44 @@
-software.
-
-## Our Vision
-
-We want OpenAssistant to be the single, unifying platform that all other systems
-use to interface with humans.
+> The FAQ page is available at [here](https://projects.laion.ai/Open-Assistant/docs/faq).
+
+Open Assistant (abbreviated as OA) is a chat-based and open-source assistant. The vision of the project is to make a large language model that can run on a single high-end consumer GPU.
+
+While Open Assistant is often compared to ChatGPT, Open Assistant should be more natural to interact with and capable at dealing with sensitive topics. When adapted from the conversational form, Open Assistant can interface with other third-party applications easily as well as retrieve information from databases and the Internet.
+
+There are four main areas that you can work on:
+1. Ranking responses in [open-assistant.io](https://www.open-assistant.io). You can take a look at [tasks docs section](https://projects.laion.ai/Open-Assistant/docs/tasks) for more information.
+2. Curating datasets and performing data augmentation. This includes scraping, gathering other public datasets, etc. Most of these efforts will be concentrated at [`/data/datasets`](https://github.com/LAION-AI/Open-Assistant/tree/main/data/datasets) and are documented at [here](https://projects.laion.ai/Open-Assistant/docs/data/datasets).
+3. Creating and fine-tuning Open Assistant itself. For that, you should pay special attention to [`/model`](https://github.com/LAION-AI/Open-Assistant/tree/main/inference).
+4. [open-assistant.io](https://www.open-assistant.io) dev. Take a close look at [`/website`](https://github.com/LAION-AI/Open-Assistant/tree/main/website) as well as [`/backend`](https://github.com/LAION-AI/Open-Assistant/tree/main/backend).
+
+## GitHub folders explanation
+> Do read the [developer guide](https://projects.laion.ai/Open-Assistant/docs/guides/developers) for further information.
+
+Here's a list of first-level folders at [Open Assistant's Github page](https://github.com/LAION-AI/Open-Assistant/).
+- [`/ansible`](https://github.com/LAION-AI/Open-Assistant/tree/main/ansible) - for managing the full stack using [Ansible](https://en.wikipedia.org/wiki/Ansible_(software))
+- [`/assets`](https://github.com/LAION-AI/Open-Assistant/tree/main/assets) - contains logos
+- [`/backend`](https://github.com/LAION-AI/Open-Assistant/tree/main/backend) - backend for open-assistant.io and discord bots, maybe helpful for locally test API calls
+- [`/copilot`](https://github.com/LAION-AI/Open-Assistant/tree/main/copilot) - read more at AWS's [Copilot](https://aws.github.io/copilot-cli/). And no, this is not a folder that contains something similar to OpenAI's Codex.
+- [`/data`](https://github.com/LAION-AI/Open-Assistant/tree/main/data) - contains [`/data/datasets`](https://github.com/LAION-AI/Open-Assistant/tree/main/data/datasets) that contains datasets in each folder, including the human response data from open-assistant.io
```

Comment:
It's not currently true that this directory contains any human response data from the site as far as I know. The directory contains scraping/parsing code for data from public sources and links to the produced datasets on HuggingFace